### PR TITLE
Initial SVM40 support

### PR DESCRIPTION
**Pull request by rnestler**

### CODE REVIEW

**Review by psachs — APPROVED** [review]

File: svm40/svm40.h [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+ * @note This command is only available in measurement mode.
+ * @return  NO_ERROR on success, an error code otherwise
+ */
+int16_t svm40_stop_measurement(void);
+
+/**
+ * Read the current measured values.
+ *
+ * The firmware updates the measurement values every second. Polling data
+ * faster  will return the same values. The first measurement is available 1
```
Comment (abrauchli) [inline svm40/svm40.h:111]:
```suggestion
 * faster will return the same values. The first measurement is available one
```

File: svm40/svm40.h [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+ * second after the start emasurement command is issued. Any readout prior to
+ * this will return zero initialized values.
+ *
+ * @note Only availabe in measurement mode.
+ * @param   voc_index           VOC algorithm output with a scaling value of 10.
+ * @param   relative_humidity   Compensated ambient humidity in %RH with a
+ *                              scaling factor of 100.
+ * @param   temperature         Compensated ambient temperature in degree
+ *                              celsius with a scaling factor of 200.
```
Comment (abrauchli) [inline svm40/svm40.h:120]: uugh, that's new - does anything speak against using the same temperature_celsius in millidegrees (with scaling factor 1000)? That would then make it compatible with the unit conversion functions from embedded-sht

**Review by abrauchli — CHANGES_REQUESTED** [review]

File: svm40/svm40.h (hunk 2 above)
Comment (rnestler) [reply svm40/svm40.h:120]: Well it is how the sensor returns the data. Should we provide conversion functions?

Comment (rnestler) [reply svm40/svm40.h:120]: I'd like to keep the low level functions of the driver as close as possible to the command specification.

**Pull request merged**